dxr/app.py

```diff
@@ -338,7 +338,7 @@ def _browse_folder(tree, path, config):
              f.get('size'),
              url_for('.browse', tree=tree, path=f.get('link', f['path'])[0]),
              f.get('is_binary', [False])[0])
-            for f in files_and_folders])
+            for f in files_and_folders if 'path' in f])
```

Comment:
When were you running into this problem? I don't see how the FileToIndex in core.py would avoid its `yield 'path', self.path` line.